### PR TITLE
Turn static NPCs and z-levels on by default

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Interface "Turns static/scenario NPCs and 'experimental' z-levels on by default"```

#### Purpose of change
The current experimental version moving towards 0.E is defined largely by improvements to NPC AI and Z-level handling. These features should be enabled by default now that they're functional for play, and to encourage playtesting.

#### Describe the solution
Enables both features by default and updates the description of z-levels.

#### Describe alternatives you've considered
This does not enable dynamic NPC spawns by default, as those are still a ways from being functional.

#### Additional context
I don't have a compiler so I can't playtest this. It's hard to imagine what could go wrong with a simple config change... but those are some famous last words.

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -1917,8 +1917,8 @@ void options_manager::add_options_world_default()
     mOptionsSort["world_default"]++;
 
     add( "ZLEVELS", "world_default", translate_marker( "Experimental z-levels" ),
-         translate_marker( "If true, experimental z-level maps will be enabled.  This feature is not finished yet and turning it on will only slow the game down." ),
-         false
+         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive lag." ),
```
Comment (DracoGriffin) [inline src/options.cpp]:
```suggestion
         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive slowdown." ),
```
Lag is a more misnomer, whereas slowdown is a bit more professionally accepted, I believe.

Comment (I-am-Erk) [reply src/options.cpp]: Sure, works for me

File: src/options.cpp [hunk 2]
```diff
@@ -1917,7 +1917,7 @@ void options_manager::add_options_world_default()
     mOptionsSort["world_default"]++;
 
     add( "ZLEVELS", "world_default", translate_marker( "Experimental z-levels" ),
-         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive lag." ),
+         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive slowdown." ),
```
Comment (HeShallNot) [inline src/options.cpp]: double space between sentences

**Comment by AMurkin** [comment]
Jenkins rebuild

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/latest-experimental-features/5582/1106

**Pull request merged**